projects/opendr_ws/src/perception/scripts/object_tracking_3d_ab3dmot.py

```diff
@@ -125,6 +156,11 @@ def callback(self, data):
         detector=detector,
         device=device,
         input_point_cloud_topic=input_point_cloud_topic,
+        output_detection3d_topic=output_detection3d_topic,
+        output_tracking3d_id_topic=output_tracking3d_id_topic,
     )
-    # begin ROS communications
-    rospy.spin()
+
+    ab3dmot_node.listen()
```

Comment:
Similar to other nodes, the `listen` method is missing as well as `init_node`. Also the comment on other nodes regarding subscriber initialization is also applied here.

Reply:
Updated